documents/research/datetime.md

```diff
@@ -1,10 +1,10 @@
 # Rust Date and Time API Request
 
-In ICU4X 0.1 we introduce a `DateTimeFormat` API for formatting date and time into human readable formats. That API is shaped after [`ICU4C`](http://userguide.icu-project.org/formatparse/datetime), [`ICU4J`](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/DateFormat.html) and [`ECMA-402`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat).
+In ICU4X 0.1 we introduce a `DateTimeFormat` API for formatting date and time into human readable formats. That API is shaped after [`ICU4C`](http://userguide.icu-project.org/formatparse/datetime), [`ICU4J`](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/DateFormat.html) and [`ECMA-402`](https://developer.mozilla.org/en-US/documents/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat).
```

Comment:
`docs`

A lot of these are on developer.mozilla.org. I won't leave any more comments for these. Please do a global search and replace.